hecuba_py/src/hdict.py

```diff
@@ -231,7 +231,7 @@ def _store_meta(storage_args):
             raise ex
 
     def __init__(self, name=None, primary_keys=None, columns=None, tokens=None,
-                 storage_id=None, indexed_on=None, **kwargs):
+                 storage_id=None, indexed_on=None, built_remotely=False, **kwargs):
```

Comment:
You also need to update  the attribute:
args_names = ["name", "primary_keys", "columns", "tokens", "storage_id", "indexed_on", "class_name"]
and add "built_remotely" there. Otherwise it may break things

Reply:
(This happens in all Hecuba Classes (Storageobj, Storagedict...)